wasm/tools/fixtures/modules.py

```diff
@@ -73,23 +78,31 @@ def spectest__print(store, arg):
             FuncType((ValType.f64, ValType.f64), ()),
             FuncType((), ()),
         ],
-        "funcaddrs": [0, 1, 2, 3, 4, 5, 6],
-        "tableaddrs": [0],
-        "memaddrs": [0],
-        "globaladdrs": [0, 1, 2],
+        "funcaddrs": [
```

Comment:
Maybe `list(map(FuncIdx, range(7)))`?

Maybe that's too much, instead: `list(map(FuncIdx, [0, 1, 2, 3, 4, 5, 6]))`

Reply:
Went with a list comprehension.  I feel icky ignoring all this mutable *everything*